test/Prosit/test_Prosit_2023_intensity_XL_CMS2_core.py

```diff
@@ -0,0 +1,72 @@
+from test.server_config import SERVER_GRPC, SERVER_HTTP
+import tritonclient.grpc as grpcclient
+import numpy as np
+from pathlib import Path
+import requests
+
+# To ensure MODEL_NAME == test_<filename>.py
+MODEL_NAME = Path(__file__).stem.replace("test_", "")
+print(MODEL_NAME)
+
+
+def test_available_http():
+    req = requests.get(f"{SERVER_HTTP}/v2/models/{MODEL_NAME}", timeout=1)
+    assert req.status_code == 200
+
+
+def test_available_grpc():
+    triton_client = grpcclient.InferenceServerClient(url=SERVER_GRPC)
+    assert triton_client.is_model_ready(MODEL_NAME)
+
+
+def test_inference():
+    seq_1 = np.load(
+        "/workspace/koina/test/Prosit/arr_Prosit_2023_intensity_XL_CMS2_seq_1.npy",
```

Comment:
Adjust the path to start with `test/` at the moment your tests only work in the devcontainer.